### PR TITLE
Document removal instructions in the README

### PR DESCRIPTION
**Pull request by whereisaaron**
Here's a starter for a removal section, discussed in #162. Modify it however you like.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,4 +96,30 @@ make
...
+After a couple of minutes you can go ahead and remove the operator itself.
+
+```
+kubectl delete -f deployment.yaml
+```
+
+The operator automatically creates two services and defines three third party resources. You can clean these up now.
+
+```
+kubectl delete --ignore-not-found service prometheus-operated alertmanager-operated
```
Comment (brancz) [inline README.md]: these are created in each namespace and should then also be removed from all namespaces

File: README.md [hunk 2]
```diff
@@ -96,4 +96,30 @@ make
 hack/run-external.sh <kubectl cluster name>
 ```
 
+## Removal
+
+To remove the operator and prometheus, first delete any third party resources you created. The
+operator will automatically shut down and remove prometheus and alertmanager pods, and associated configmaps.
```
Comment (brancz) [inline README.md]: Prometheus and Alertmanager capitalized

File: README.md (hunk 1 above)
Comment (brancz) [inline README.md]: I think the operator should cleanup these actually, but this is applicable until we implemented that.

File: README.md (hunk 2 above)
Comment (whereisaaron) [inline README.md]: Capitalized

File: README.md (hunk 1 above)
Comment (whereisaaron) [inline README.md]: Updated examples to delete from all namespaces.

**Comment by brancz** [comment]
lgtm :+1: thanks!

**Pull request merged**